### PR TITLE
Add EMR Connectivity Doc

### PR DESCRIPTION
**Pull request by isuruuy429**
This PR includes the following. 
- EMR Systems Overview
- EMR Connector use case for EPIC (Ballerina)

### CODE REVIEW

File: en/docs/emr-connectivity/guides/emr-connector.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    openssl genrsa -out <path_to_key>/privatekey.pem 2048
+    ```
+
+    Creating the public key
+    ```
+    openssl req -new -x509 -key <path_to_key>/privatekey.pem -out <path_to_key>/publickey509.pem -subj '/CN=myapp'
+    ```
+
+2. Login to [EPIC FHIR](https://fhir.epic.com/) with your credentials. 
+3. Navigate to 'Build Apps' tab to create an application.
```
Comment (sameeragunarathne) [inline en/docs/emr-connectivity/guides/emr-connector.md:21]: Let's link the epic documentation on creating apps

File: en/docs/emr-connectivity/guides/emr-connector.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+## Overview
+The FHIR Client Connector is a versatile tool designed to streamline integration with healthcare systems that adhere to the FHIR standard. Whether you're using Ballerina or a Micro Integrator, this connector provides a unified interface to interact with FHIR-compliant APIs, making it easier to build and manage healthcare applications.
```
Comment (sameeragunarathne) [inline en/docs/emr-connectivity/guides/emr-connector.md]: This sentence seems to be vague and unclear, fhir client connector is used to connect to FHIR servers and repositories.

File: en/docs/emr-connectivity/guides/emr-connector.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+## Connecting with Epic EMR 
+In this example we will look into the steps how can we use the Client Connector to connect with the Epic EMR system.
```
Comment (sameeragunarathne) [inline en/docs/emr-connectivity/guides/emr-connector.md:5]:
```suggestion
In this example we will look into the steps how can we use the Client Connector to connect with the Epic FHIR sandbox. 
```

File: en/docs/emr-connectivity/guides/emr-connector.md (hunk 2 above)
Comment (isuruuy429) [reply en/docs/emr-connectivity/guides/emr-connector.md]: Fixed.

File: en/docs/emr-connectivity/guides/emr-connector.md (hunk 1 above)
Comment (isuruuy429) [reply en/docs/emr-connectivity/guides/emr-connector.md:21]: Fixed.

**Review by sameeragunarathne — APPROVED** [review]

**Pull request merged**